### PR TITLE
docs: update faqs, intro and common issues

### PR DESCRIPTION
**Pull request by aavshr**
- update faqs and about for micro
- update general common issues and faqs

### CODE REVIEW

File: docs/common_issues.md [hunk 1]
```diff
@@ -34,4 +34,12 @@ BINARY_CONTENT_TYPES=image/*,font/*
 
 will let the micro serve these files. 
 
-Please, use the [`deta update -e [env_file_name]`](https://docs.deta.sh/docs/cli/commands#deta-update) command to update the environment variables of the micro.
\ No newline at end of file
+Please, use the [`deta update -e [env_file_name]`](https://docs.deta.sh/docs/cli/commands#deta-update) command to update the environment variables of the micro.
```
Comment (xeust) [inline docs/common_issues.md]: Please, use the [`deta update -e [env_file_name]`](https://docs.deta.sh/docs/cli/commands#deta-update) command to update the environment variables of the Micro.

File: docs/common_issues.md [hunk 2]
```diff
@@ -34,4 +34,12 @@ BINARY_CONTENT_TYPES=image/*,font/*
...
+
+### Cannot login from the web browser after `deta login`
+
+Logging in with `deta login` from the [Brave Browser](https://brave.com/) does not work at the moment. Please use a different browser to login (you can copy the link in a different browser after the login page opens up), or use [deta access tokens](https://docs.deta.sh/docs/cli/auth#deta-access-tokens) instead.
+
+### Account is unconfirmed but did not receive a confirmation email
```
Comment (xeust) [inline docs/common_issues.md]: ### Account is unconfirmed but I did not receive a confirmation email

File: docs/faqs.md [hunk 3]
```diff
@@ -4,18 +4,14 @@ title: Frequently Asked Questions
...
 
 2. **API Keys**: This is an optional key to protect your Micro HTTP endpoint or to implement client-based rules. Creating API Keys from the Deta CLI is described [here](/docs/cli/commands#deta-auth-create-api-key). Read more about API Keys [here](https://en.wikipedia.org/wiki/Application_programming_interface_key). If this key is leaked, your micro endpoint can be used by unwanted clients.
 
-3. **Access Tokens**: This token lets other programs manage (create, modify & delete) your Micros, Projects and Bases on your behalf. It's mainly used to let you have programmatic access to our API. Use cases: CI/CD, Gitpod, GitHub Actions. Information on Deta Access Tokens can be found [here](/docs/cli/auth#deta-access-tokens). If this key is leaked, your whole Deta account is compromised.
+3. **Access Tokens**: This token lets other programs manage (create, modify & delete) your Micros, Projects, Bases and Drives on your behalf. It's mainly used to let you have programmatic access to our API. Use cases: CI/CD, Gitpod, GitHub Actions. Information on Deta Access Tokens can be found [here](/docs/cli/auth#deta-access-tokens). If this key is leaked, your whole Deta account is compromised.
```
Comment (xeust) [inline docs/faqs.md]: **Access Tokens**: This token lets you interact with Deta via the CLI from a remote environment. Use cases: CI/CD, Gitpod, GitHub Actions. Information on Deta Access Tokens can be found [here](/docs/cli/auth#deta-access-tokens). If this key is leaked, your whole Deta account is compromised.

^ or something like this? E.g. not mentioning explicitly they can hit any endpoint and use Deta as Infra-as-code?

Comment (aavshr) [reply docs/faqs.md]: will update thanks

File: docs/common_issues.md [hunk 4]
```diff
@@ -34,4 +34,12 @@ BINARY_CONTENT_TYPES=image/*,font/*
...
 
-Please, use the [`deta update -e [env_file_name]`](https://docs.deta.sh/docs/cli/commands#deta-update) command to update the environment variables of the micro.
\ No newline at end of file
+Please, use the [`deta update -e [env_file_name]`](https://docs.deta.sh/docs/cli/commands#deta-update) command to update the environment variables of the Micro.
+
+### Cannot login from the web browser after `deta login`
+
+Logging in with `deta login` from the [Brave Browser](https://brave.com/) does not work at the moment. Please use a different browser to login (you can copy the link in a different browser after the login page opens up), or use [deta access tokens](https://docs.deta.sh/docs/cli/auth#deta-access-tokens) instead.
+
+### Account is unconfirmed but I did  not receive a confirmation email
```
Comment (xeust) [inline docs/common_issues.md]: ### Account is unconfirmed but I did not receive a confirmation email

**Review by xeust — APPROVED** [review]

**Pull request merged**